theano/gpuarray/dnn.py

```diff
@@ -354,7 +354,9 @@ class GpuDnnConvDesc(COp):
     See the doc of :func:`dnn_conv` for a description of the parameters
 
     """
-
+    
+    prec_GpuDnn = get_precision(precision, [img, kerns])
```

Comment:
@nouiz , will replacing with the default precision (in below line) work?

`precision = `theano.config.dnn.conv.precision``